486_Project/Characters_Class.py

```diff
@@ -46,8 +52,12 @@ def __init__(self):
     def new_max_hp(self):
         self.warrior_max_hp = self.stats[3] * 4
 
+    # function for when player gets new weapon
+    def new_weapon(self):
+        pass
+
 
-class bad_warrior:
+class Bad_warrior:
```

Comment:
Is this a sub-class of Warrior, inheriting all the attributes and adding a few extra?